tensorboard/webapp/metrics/views/card_renderer/run_name_component.ts

```diff
@@ -14,12 +14,20 @@ limitations under the License.
 ==============================================================================*/
 import {ChangeDetectionStrategy, Component, Input} from '@angular/core';
 
+import {ExperimentAlias} from '../../../experiments/types';
+
 @Component({
   selector: 'card-run-name-component',
-  template: '{{ name }}',
+  template: `<tb-experiment-alias
+      *ngIf="experimentAlias != null"
+      [alias]="experimentAlias!"
+    ></tb-experiment-alias>
+    <span *ngIf="experimentAlias != null">/</span>
+    <span>{{ name }}</span>`,
   styleUrls: ['run_name_component.css'],
   changeDetection: ChangeDetectionStrategy.OnPush,
 })
 export class RunNameComponent {
   @Input() name!: string;
```

Comment:
Instead of `name` and `experimentAlias` as Inputs, I am thinking whether we should just pass `Experiment` and `Run` as Input. WDYT?

Reply:
No action required: since it is literally "RunName" component, I think knowing about `Run` is not too wild. The suggestion was not to make it so that the component gets more information but rather I did not want the consumer of the widgets to carefully destruct information readily available on Run and Experiment to conform to this Input.